docs/content/status.md

```diff
@@ -68,6 +135,7 @@ Status updates contain the following fields:
 | `discovery.active_revision` | `string` | Opaque revision identifier of the last successful discovery activation. |
 | `discovery.last_successful_download` | `string` | RFC3339 timestamp of last successful discovery bundle download. |
 | `discovery.last_successful_activation` | `string` | RFC3339 timestamp of last successful discovery bundle activation. |
+| `metrics` | `array` | Prometheus metrics as return by the Gather method. See [Prometheus Go client](https://github.com/prometheus/client_golang/blob/v1.1.0/prometheus/registry.go) for exact format.|
```

Comment:
nit: s/metrics as return by/metrics as returned by/

I'm also wondering if that link is the best place to send people for docs. Opening it up it isn't super easy to find the structure (relative to the other stuff we document in the payload). I'll look around a bit more for maybe a better schema or something we can use, maybe even just a godoc page for the same stuff or something.

Reply:
yep, IDK what the right URL would be. The type is defined in the protobuf-generated code. URL that points to exact line might become wrong the next time we update the prometheus client

Reply:
fixed the typo